docs/logging.md

```diff
@@ -0,0 +1,52 @@
+## Logging Congiguration
+--
+
+firecracker-containerd allows for users to specify per application/library
+logging. We do this by setting the `log_level` field in the
+firecracker-runtime.json file.
+
+```json
+{
+  "firecracker_binary_path": "/usr/local/bin/firecracker",
+  "kernel_image_path": "/var/lib/firecracker-containerd/runtime/default-vmlinux.bin",
+  "kernel_args": "ro console=ttyS0 noapic reboot=k panic=1 pci=off nomodules systemd.journald.forward_to_console systemd.unit=firecracker.target init=/sbin/overlay-init",
+  "root_drive": "/var/lib/firecracker-containerd/runtime/default-rootfs.img",
+  "cpu_count": 1,
+  "cpu_template": "T2",
+  "log_levels": "debug",
+  "jailer": {
+    "runc_binary_path": "/usr/local/bin/runc"
+  }
+}
+```
+
+| log levels                     | description                                                               |
+| :---------------------------:  | :-----------------------------------------------------------------------: |
+| debug                          | Enables default values for logging. This will set all log levels to debug |
+| firecracker_error              | Logs entries that come from Firecracker's error macro                     |
+| firecracker_warning            | Logs entries that come from Firecracker's error and warn macro            |
+| firecracker_info               | Logs entries that come from Firecracker's error, warn, and info macro     |
+| firecracker_debug              | Most verbose log level for firecracker                                    |
+| firecracker_output             | Logs Firecracker's stdout and stderr                                      |
+| firecracker-go-sdk             | Enables genrally logging for the firecracker-go-sdk                       |
+| firecracker-containerd_error   | Logs any error information during the container/vm lifecycle              |
+| firecracker-containerd_warning | Logs any error level logs along with any warn level logs                  |
+| firecracker-containerd_info    | Logs any error, warn, and info level logs                                 |
+| firecracker-containerd_debug   | Most verbose logging for firecracker-containerd                           |
+
+`log_levels` is a case insensitive list of comma seperated log levels.
```

Comment:
Does it have to be case-insensitive? Most of stuff we have is case-sensitive.

Reply:
It doesn't have to be, but again this was to support backwards compatibility, but since that is no longer an option I can make it case sensitive